### PR TITLE
trc-37-FeedGroup-component

### PR DESCRIPTION
**Pull request by willcary**
# TL/DR
This story creates a FeedGroup component which accepts a heading and an array of feedItem data as props.

FeedGroup-Component

## Overview of Changes
`FeedGroup.tsx`

- creates FeedGroupProps interface with `heading` and `feedItems`
- creates FeedGroup component that accepts `heading` and `feedItems` as props to render a heading followed by a list of `<FeedItem />` components.

`FeedGroup.scss`

- add scss file to style `FeedGroup` component.

`FeedGroup.utils.tsx`

- create mock `feedItems` data to use for Storybook and unit tests.

`FeedGroup.stories.tsx`

- adds a default story with the heading "FEEDS" and the `feedItems` array from the utils file.

`FeedGroup.test.tsx`

- Creates unit tests to ensure the heading and `FeedItem`'s are displayed properly.

`FeedItem.tsx` and `FeedItem.scss`

- Adds and styles div container around `Icon` to constrain the `Icon` width to match design specifications. 

`popularIcon.svg`

-Adds `popularIcon.svg` to the assets folder for use in the `feedItems` mock data.

### CODE REVIEW

File: apps/web/src/components/FeedGroup/FeedGroup.test.tsx
Comment (wijohnst) [inline apps/web/src/components/FeedGroup/FeedGroup.test.tsx:1]: Can you add a test for `onClick` handling?

File: apps/web/src/components/FeedGroup/FeedGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const FeedGroup = ({ heading, feedItems }: FeedGroupProps) => {
+  return (
+    <div className="feedbar-group">
+      <h3 className="feedbar-heading">{heading}</h3>
+      {feedItems.map((feedItem) => (
+        <FeedItem
+          svg={feedItem.svg}
+          label={feedItem.label}
+          href={feedItem.href}
+          key={feedItem.label}
```
Comment (wijohnst) [inline apps/web/src/components/FeedGroup/FeedGroup.tsx:27]: Can we come up with a `key` value that is more unique than label? If, for some reason, we had 2 `FeedItem` components with the same `label` value this could cause a problem.

File: apps/web/src/components/FeedGroup/FeedGroup.utils.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    svg: <HomeIcon />,
+    label: 'Home',
+    href: '',
+  },
+  {
+    svg: <PopularIcon />,
+    label: 'Popular',
+    href: '',
+  },
+];
```
Comment (wijohnst) [inline apps/web/src/components/FeedGroup/FeedGroup.utils.tsx:16]: Are we sure we want to hardcode these values? I feel like the `feedItems ` would be returned from an API call.

File: apps/web/src/components/FeedItem/FeedItem.scss [hunk 3]
```diff
@@ -12,6 +12,10 @@ a:-webkit-any-link {
   height: 100%;
 }
 
+.feedsbar-feeditem-icon-container {
+  width: 35px;
+}
+
```
Comment (wijohnst) [inline apps/web/src/components/FeedItem/FeedItem.scss:18]: Same comment re: relative units

**Review by wijohnst — COMMENTED** [review]
Great start on this. Left a few comments.

File: apps/web/src/components/FeedGroup/FeedGroup.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
+.feedbar-group {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 10px;
```
[the remark this answers is not in the document]
Comment (willcary) [reply apps/web/src/components/FeedGroup/FeedGroup.scss:5]: For sure, I generally avoid px myself, but that was what the figma css had so I went with it. Good to know for the future!

File: apps/web/src/components/FeedGroup/FeedGroup.test.tsx
Comment (willcary) [reply apps/web/src/components/FeedGroup/FeedGroup.test.tsx:1]: Since the `FeedItem`s do not have an `onClick` function (just an anchor tag `href` attribute), it seems like `onClick` testing isn't feasible. I spent a while trying to figure out how to do it, but everything I saw involved adding an `onClick` attribute to the `FeedItem`, which I thought would be a bit much. I did add tests to ensure the `href` is passed properly to each `FeedItem`. Let me know what you think.

**Review by wijohnst — APPROVED** [review]
